site/source/user-manual/service-guides/lightning/index.rst

```diff
@@ -11,14 +11,33 @@ The following guide will get you up and running with a lightning node and walk y
 
   lightning-first-channel
 
-Now that you're up and running, it's time to connect some wallets to your node so that you can use lightning from anywhere!
+Now that you're up and running, it's time to connect a wallets to your node so that you can operate your node and use lightning to make and receive payments from anywhere!
 
+If you're using Core Lightning (CLN), you can select one of the following options:
 
+  .. toctree::
+    :maxdepth: 2
 
-.. toctree::
-  :maxdepth: 2
+    connecting-cln
+
+If you're using LND, you can select one of the following options:
+
+  .. toctree::
+    :maxdepth: 2
+
+    connecting-lnd
+
+If you would like to connect to LND via LND Connect (LNC) you can do so with Alby and Zeus via the Lightning Terminal service:
```

Comment:
```suggestion
If you would like to connect to LND via LND Connect (LNC) in order to avoid using Tor, you can do so with Alby and Zeus via the Lightning Terminal service:
```